view_products.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Products Available</title>
+</head>
+
+<body background="image/images.jpg">
+    <nav class="topnav">
+        <ul>
+            <li><a href="view_products.html">Products</a></li>
+            <li><a href="sales.html">Sales</a></li>
+            <li><a href="add.html">Add</a></li>
+            <li><a href="index.html">Logout</a></li>
+        </ul>
+    </nav>
+    <main class="boxed">
+        <table>
+            <thead>
+                <th><b>Product</b></th>
+                <th><b>Brand</b></th>
+                <th><b>Price</b></th>
+                <th><b>Quantity</b></th>
+                <th></th>
+            </thead>
+            <tbody>
+                <td>Feminine watches</td>
+                <td>Rolex</td>
+                <td>$170</td>
+                <td>50 pieces</td>
+                <td>
+                    <li><a href="view_one.html"><span class="black">view</span></a></li>
+                </td>
+            </tbody>
```

Comment:
These should be table row tags `tr` and not `tbody`. The `tbody` tag should enclose all rows that aint part of the the table header